### ISSUE TITLE
Cannot read accessibility instructions.

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#177899]
Testing #177674

![image](https://user-images.githubusercontent.com/3840081/226743733-9ac431ad-95e3-43fb-a9da-eb30449723d5.png)

**Comment by meganrogge** [comment]
since this is targeted at screen reader users, it's not too important if the text is all visible imo

**Comment by Tyriar** [comment]
I think we should do this as it should be fairly easy to fix with a scroll bar and low vision users may also use a screen reader.

**Comment by meganrogge** [comment]
The issue is even if I add scrolling, it's off screen. I think we should use `widgetManager.attachWidget` here.

**Comment by meganrogge** [comment]
I also think we should use an editor so a screen reader user can use arrow keys to navigate

**Comment by meganrogge** [comment]
To also fix #173911, let's use something similar to the accessible buffer here, but with the help content
